.github/workflows/backup.yml

```diff
@@ -0,0 +1,40 @@
+name: Backup and Delete Old Entries
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
@riderx do you want this as a cron job?

Reply:
or something that you can click on and trigger?

Reply:
Pinged Martin on Discord.

Reply:
as cron job is good

Reply:
@ayusham001 feel free to close this review then as well. great work!